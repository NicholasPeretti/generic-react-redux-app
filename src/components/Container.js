import Style from '../style'
import styled from 'styled-components'

const Container = styled.div`
  margin: 0px ${Style.spacing.padding3};
`

export default Container
