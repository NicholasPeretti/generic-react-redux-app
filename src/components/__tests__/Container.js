import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import Container from '../Container'

describe('Container', () => {
  PresentationalTest(Container)

  test('Should match snapshot', () => {
    const wrapper = shallow(<Container>This is a Container</Container>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
