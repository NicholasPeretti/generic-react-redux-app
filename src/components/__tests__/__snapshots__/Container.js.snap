// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Should match snapshot 1`] = `"<div class=\\"sc-bdVaJa lhlgbq\\">This is a Container</div>"`;
